### DISCUSSION TITLE
Welcome to TechArch Discussions!

### DISCUSSION THREAD

**Discussion by elisatrevino** [trustoverip/TechArch#48] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by talltree** [comment]
@elisatrevino Thank you for turning Discussions on. This will help us separate posts that are not actually issues with [ToIP Technology Architecture V1.0 Specification](https://github.com/trustoverip/TechArch/blob/main/spec.md) from [issues](https://github.com/trustoverip/TechArch/issues) that directly impact the content, formatting, or publishing of the spec (especially heading towards our deadline of publishing the First Public Review Draft by the first day of [Internet Identity Workshop](http://www.internetidentityworkshop.com/) on November 15th).
